src/ConnectedMode/Shared/SharedBindingConfigFileProvider.cs

```diff
@@ -65,7 +64,8 @@ public SharedBindingConfigModel ReadSharedBindingConfigFile(string filePath)
 
                 if (result.IsSonarCloud())
                 {
-                    result.Uri = SonarCloudUri;
+                    result.Region = string.IsNullOrEmpty(result.Region) ? CloudServerRegion.Eu.Name : result.Region;
```

Comment:
Please extract the stored region in a variable and used it to fill both the Region and Url properties
```suggestion
var region = CloudServerRegion.GetRegionByName(result.Region);
                    result.Region = region.Name
```